internal-ui/src/pages/DashboardPage.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Breadcrumb,
+    BreadcrumbItem,
+    PageSection,
+    Grid,
+    GridItem
+} from "@patternfly/react-core";
+import {Link} from "react-router-dom";
+import {AverageNumberofConversations, AverageUserStatistics, NumberOfSessions, SelectFilters, SessionReviewCount, UniqueUsers} from "../components/DashboardComponent.tsx";
```

Comment:
Looks like you aren't using the UniqueUsers comp, just need to add it (or remove it).